src/components/Article.js

```diff
@@ -17,9 +18,9 @@ export default () => {
         <article className=${styles.container}>
           <div className=${styles.welcome}>
             <a className=${styles.formidaLogo} href="https://formidable.com">
-              ${FormidableLogo}
+              <${FormidableLogo} />
```

Comment:
Components now converted to JSX can no longer be called in the shorthand manner.